_includes/header.html

```diff
@@ -16,10 +16,10 @@
     </li>
     <li class="col-xs-12 col-md-10 menu">
       <nav aria-label="Banner links">
-        <h2><a href="/GettingStarted">Getting Started</a></h2>
+        <h2><a href="/Resources">Resources</a></h2>
         <h2><a href="/processes">Contributing</a></h2>
         <h2><a href="/projects">Projects</a></h2>
-        <h2><a href="/Tutorials">Tutorials</a></h2>
+        <h2><a href="/Tutorials/">Tutorials</a></h2>
```

Comment:
I wonder, should these go through ` | relative_url ` to work properly in the Github preview?

Reply:
Like this?

```
        <h2><a href="{{ relative_url }}/Resources" />Resources</h2>
        <h2><a href="{{ relative_url }}/processes" />Contributing</a></h2>
        <h2><a href="{{ relative_url }}/Projects" />Projects</h2>
        <h2><a href="{{ relative_url }}/Tutorials" />Tutorials</h2>
```

Reply:
The syntax goes like this:
```
<h2><a href="{{ '/Resources' | relative_url }}" />Resources</h2>
```

Reply:
Thanks!